### PR TITLE
Detect links that should go to docs.cozy.io

### PR DESCRIPTION
**Pull request by ptbrowne**
- Run remark on our docs to lint and fix them
- Use an adhoc remark plugin to detect and fix links that should go
  to docs.cozy.io

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "remark-cli": "^7.0.0",
     "remark-frontmatter": "^2.0.0",
+    "remark-lint-no-dead-urls": "^1.1.0",
```
Comment (Crash--) [inline package.json:11]: this should not be part of this commit, but promising 👏

File: remark/detect-bad-links.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+const updateURLToDocsCozyIO = url => {
+  const [base, fragment] = url.split('#')
+  const newURL = base
+    .replace('https://github.com/cozy/', 'https://docs.cozy.io/en/')
+    .replace('/blob/master', '')
+    .replace(/\.md$/, '')
+    .replace('cozy-stack/docs', 'cozy-stack')
+    .replace('cozy-konnector-libs/packages/cozy-konnector-libs/docs', 'cozy-konnector-libs')
```
Comment (Crash--) [inline remark/detect-bad-links.js:16]: don't you need to handle /README/ ?

**Review by Crash-- — APPROVED** [review]
👏 👏 👏 👏

File: remark/detect-bad-links.js (hunk 2 above)
Comment (ptbrowne) [reply remark/detect-bad-links.js:16]: Yes, I did that by hand afterwards, fixing..

File: package.json (hunk 1 above)
Comment (ptbrowne) [reply package.json:11]: Mmh I added it since it was useful to detect dead urls, but I do not want to run it in the CI as it takes a bit of time.
